.prettierignore

```diff
@@ -33,3 +33,6 @@ public/openapi3.json
 
 # Generated Kinds report
 kinds/report.json
+
+# Generated schema docs
+docs/sources/developers/kinds/
```

Comment:
I kept getting `prettier` warning reg table formatting. Disabling the prettier for the time being, till I find a better way to render tables in markdown (thus far github.com/olekukonko/tablewriter is being used for that).

This is my diff after running `yarn run prettier:write` on generated docs:
<img width="514" alt="Screenshot 2023-01-10 at 17 15 08" src="https://user-images.githubusercontent.com/10127682/211604102-29627a99-567b-4ede-bdfe-69d5561e03ff.png">

Reply:
Disabling linters on generated artifacts is a common and acceptable practice 👍🏻

Reply:
Thanks for confirming!